js/api.ts

```diff
@@ -193,6 +189,7 @@ export class API<T extends APIObject> {
         let resource = this.Resource;
         return this.client.request("/api/" + resource, "POST", JSON.stringify(obj), {})
             .then(function (data) {
+                console.log(data)
```

Comment:
I am not sure we want to leave this `console.log()`

Reply:
yep, added for debugging purpose and forgot to remove.